test('hello', () => {
  console.log('');
});
test('this should fail', () => {
  throw new Error('fail');
});
